src/Plateau.php

```diff
@@ -1,137 +1,92 @@
 <?php
- echo "<style>
-    .geo{
-        width:50px;
-        height:50px;
-        border: solid 1px black;
-        background-color : cyan;
-    }
-    .hist{
-        width:50px;
-        height:50px;
-        border: solid 1px black;
-        background-color : yellow;
-    }
-    .diver{
-        width:50px;
-        height:50px;
-        border: solid 1px black;
-        background-color : pink;
-    }
-    .sport{
-        width:50px;
-        height:50px;
-        border: solid 1px black;
-        background-color : orange;
-    }
-    .info{
-        width:50px;
-        height:50px;
-        border: solid 1px black;
-        background-color : green;
-    }
-    .perso{
-        width:50px;
-        height:50px;
-        border: solid 1px black;
-        background-color : purple;
-    }
-    .depart{
-        width:100px;
-        height:100px;
-        border: solid 1px black;
-        background-color : black;
-    }
-    table{
-        background:grey;
-    }
- </style>";
+namespace trivial;
 
- $taille_tab=13;
-$tableau = array();
+ class Plateau {
+    private $tableau = array();
+    private $taille_tab=13;
 
-for($i=1;$i<=$taille_tab;$i++){
-    for($j=1;$j<=$taille_tab;$j++){
-        if($i-0.5==$taille_tab/2 && $j-0.5==$taille_tab/2 ){
-            $tableau[$i][$j]="depart";
-        }
-        else
-        if($i==1 || ($i-0.5==$taille_tab/2 && $j<$taille_tab/2)){
+    public function creaPlateau(){
+    for($i=1;$i<=$this->taille_tab;$i++){
```

Comment:
This is hardly readable, some clues:
- Can you write this code in another way? Maybe just initialize the table without any for loops (the table doesn't change size right?)
- Can you try to externalize some of the code to named functions?
- Can you take more care with variable naming? `$i` and `$j` usually can be replaced with things like `$column` and `$row`